scripts/exglobal_atmos_analysis_calc.sh

```diff
@@ -115,53 +116,69 @@ rm -rf dir.*
 ##############################################################
 # If analysis increment is written by GSI, produce an analysis file here
 if [[ "${DO_CALC_ANALYSIS}" == "YES" ]]; then
-   # link analysis and increment files
-   ${NLN} "${ATMANL}" siganl
-   ${NLN} "${ATMINC}" siginc.nc
-   if [[ "${DOHYBVAR}" == "YES" && "${l4densvar}" == ".true." && "${lwrite4danl}" == ".true." ]]; then
-      ${NLN} "${ATMA03}"   siga03
-      ${NLN} "${ATMI03}"   sigi03.nc
-      ${NLN} "${ATMA04}"   siga04
-      ${NLN} "${ATMI04}"   sigi04.nc
-      ${NLN} "${ATMA05}"   siga05
-      ${NLN} "${ATMI05}"   sigi05.nc
-      ${NLN} "${ATMA07}"   siga07
-      ${NLN} "${ATMI07}"   sigi07.nc
-      ${NLN} "${ATMA08}"   siga08
-      ${NLN} "${ATMI08}"   sigi08.nc
-      ${NLN} "${ATMA09}"   siga09
-      ${NLN} "${ATMI09}"   sigi09.nc
-   fi
-   # link guess files
-   ${NLN} "${ATMG03}" sigf03
-   ${NLN} "${ATMGES}" sigf06
-   ${NLN} "${ATMG09}" sigf09
-
-   if [[ -f "${ATMG04}" ]]; then
-       ${NLN} "${ATMG04}" sigf04
-   fi
-   if [[ -f "${ATMG05}" ]]; then
-       ${NLN} "${ATMG05}" sigf05
-   fi
-   if [[ -f "${ATMG07}" ]]; then
-       ${NLN} "${ATMG07}" sigf07
-   fi
-   if [[ -f "${ATMG08}" ]]; then
-       ${NLN} "${ATMG08}" sigf08
-   fi
-
-   # Link hourly backgrounds (if present)
-   if [[ -f "${ATMG04}" && -f "${ATMG05}" && -f "${ATMG07}" && -f "${ATMG08}" ]]; then
-      nhr_obsbin=1
-   fi
-
-   ${CALCANLPY}
-   export err=$?
-   if [[ ${err} -ne 0 ]]; then
-      err_exit "Failed to run ${CALCANLPY}"
-   fi
+  ##############################################################
+  # If aerosol increment is written by GCDAS, produce aerosol analysis in atmos analysis file
+    if [[ "${RUN}" == "gcdas" ]]; then
+        # link analysis and increment files
+        ${NLN} "${ATMANL}" siganl
+        ${NLN} "${AEROINC}" siginc.nc
```

Comment:
how about we just change what ATMINC is depending on the RUN? and then we don't need all of this added to an if-else? There's no harm in linking sigf03 and sigf09 is there?